### DISCUSSION TITLE
Database wont setup?

### DISCUSSION THREAD

**Discussion by AlmonPista** [DSJAS/DSJAS#58] [Installation help] [answered]
After filling out the database information, host name, database name, user and password, the test button says it was successful, but it will not go to the next step. The guide says it can take time so I left it a few hours and still nothing. What do now?

**Comment by ejv2** [comment, marked as answer]
Can you please post what PHP version you are using and what version of DSJAS you downloaded?

**Reply by AlmonPista** [reply]
PHP version: 8.1.6, 
DSJAS - release - alpha 
and using XAMPP

**Reply by ejv2** [reply]
Please bear with me. I will do some testing once I have the chance. There are some known issues with PHP 8 support on older releases. If you want an immediate fix, try using an older release of PHP, as PHP releases are not backward compatible. DSJAS was designed for PHP 7.

Nevertheless, I will attempt to track down the source of issues with the installer, as a few have been rolling in recently.

**Reply by AlmonPista** [reply]
that worked thank you for the help and fast response
